### PR TITLE
Deprecate Expanse (v1) Pack + add commands to v2 (from v1)

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34162

## Description
- Added the following command.
  - ***expanse-get-domains-for-certificate***: Returns all domains which have been seen with the specified certificate.
- Upgraded the Docker image to demisto/python3:3.9.2.18414.
- Added UTs
- Added relevant tests for those commands to TPB (v2).
- Modified Readme file.
-   - ***expanse-get-exposures***: Returns exposure information about the IP. Is deprecated by Expanse during this month, specified in the yml file of v1.

## Minimum version of Demisto
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by guykeller — APPROVED** [review]

File: Packs/ExpanseV2/.secrets-ignore [hunk 1]
```diff
@@ -145,4 +145,9 @@ luigi@cortexlab.local
...
 95.114.16.51
 44.224.0.0
-44.224.1.1
\ No newline at end of file
+44.224.1.1
+12.4.49.74
+12.2.248.97
+33.2.243.123
+192.168.168.168
+beta+vandelay@expanseinc.com
```
[the remark this answers is not in the document]
Comment (abaumgarten) [reply Packs/ExpanseV2/.secrets-ignore:153]: They are all in secrets-ignore in v1 , so I assume it is ok

File: Packs/Expanse/ReleaseNotes/1_2_0.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Expanse (Deprecated)
+- This integration is now deprecated. Please use the **Expanse V2** pack instead.
```
Comment (amshamah419) [inline Packs/Expanse/ReleaseNotes/1_2_0.md:4]:
```suggestion
- This integration is now deprecated. Please use the **Expanse V2** integration instead.
```

File: Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py [hunk 3]
```diff
@@ -533,6 +534,49 @@ def parse_asset_data(self, issue: Dict[str, Any],
             changed = True
         return assets, ml_feature_list, changed
 
+    def get_exposures_ips(self, params):
```
Comment (andrew-expanse) [inline Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py:537]: The Exposures API is being deprecated, so we shouldn't include this in the new version.

File: Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py [hunk 4]
```diff
@@ -2040,6 +2086,146 @@ def get_risky_flows_command(client: Client, args: Dict[str, Any]) -> CommandResu
     )
 
 
+def exposures_command(client: Client, args: Dict[str, Any]) -> CommandResults:
```
Comment (andrew-expanse) [inline Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py:2089]: Exposures is being deprecated, so we should remove this command.

**Comment by andrew-expanse** [comment]
Hi @abaumgarten, sorry for not being involved earlier, but Exposures is being deprecated and will stop functioning later this month, so we can remove all exposure-related requests.

File: Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py [hunk 5]
```diff
@@ -533,6 +534,43 @@ def parse_asset_data(self, issue: Dict[str, Any],
             changed = True
         return assets, ml_feature_list, changed
 
+    def fetch_certificates(self, params):
```
Comment (andrew-expanse) [inline Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py:537]: I think this is duplicative of `get_certificates` above.

File: Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py [hunk 6]
```diff
@@ -533,6 +534,43 @@ def parse_asset_data(self, issue: Dict[str, Any],
...
+
+        :param params: Search parameters
+        :return: List of certificate objects
+        """
+        return self._paginate(
+            method='GET',
+            url_suffix='v2/assets/certificates',
+            params=params)
+
+    def fetch_certificate(self, md5_hash):
```
Comment (andrew-expanse) [inline Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py:549]: I believe that `get_certificate_by_md5_hash` above should do the same thing.

File: Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py [hunk 7]
```diff
@@ -533,6 +534,43 @@ def parse_asset_data(self, issue: Dict[str, Any],
...
+        :param md5_hash: Search term for certificates
+        :return: Certificate details objects
+        """
+        url = 'v2/assets/certificates/{}'.format(md5_hash)
+        return self._http_request(
+            method='GET',
+            url_suffix=url,
+            params={})
+
+    def fetch_ips(self, params):
```
Comment (andrew-expanse) [inline Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py:562]: `get_ips` above should cover this functionality.

**Review by andrew-expanse — APPROVED** [review]

**Pull request merged**